client/components/widget/query/builder/query-builder-column-config-directive.html

```diff
@@ -0,0 +1,78 @@
+<div>
+  <div class="sidebar-group-title">Columns</div>
+
+  <div class="sidebar-item">
+    <div class="sidebar-item-label">
+      <input type="checkbox"
+             class="widget-columns-show-date"
+             ng-model="ngModel.datasource.config.results.show_date">
+      show date
+    </div>
+  </div>
+
+  <div class="sidebar-item" ng-show="ngModel.datasource.config.results.show_date">
+    <div class="sidebar-item-label">
+      grouping (blank for none)
+    </div>
+    <div class="sidebar-item-value">
+      <input class="form-control widget-columns-date-group"
+         popupbox
+         popupbox-template-url="/static/components/widget/query/builder/picklist-template.html"
```

Comment:
I think this would be clearer as a constant in in the directive JS, so that future changes to the folder structure primarily change the content of that file.